src/starbase/setup.ts

```diff
@@ -83,7 +83,15 @@ async function setupIntegration(integration: StarbaseIntegration) {
   if (await isDirectoryPresent(integration.directory)) {
     await updateIntegrationDirectory(integration.directory);
   } else {
-    await Clone.clone(integration.gitRemoteUrl, integration.directory);
+    const options: CloneOptions = {
+      fetchOpts: {
+        callbacks: {
+          certificateCheck: () => 0,
```

Comment:
@ceelias I'm not sure whether we want certificate checking disabled as the default behavior. It seems that the ideal behavior would be disabling cert checks as a fallback when cert checking fails. 

cc @adam-in-ict thoughts?

Reply:
I can see your concern here. I think though, that this doesn't actually disable the check, but rather will get called in the case that the error gets bubbled up. Maybe we log that it occurred and keep it moving?

Or do you think there would be some value in exposing a config property to use this callback and point users to it in the case that they encounter this issue?

Reply:
> I think though, that this doesn't actually disable the check, but rather will get called in the case that the cert check error is encountered.

@ceelias Not sure whether that is true or not. My guess is that `certificateCheck` is a function implementation override. It's not completely clear from the documentation, but I believe that to be the case.

See: https://www.nodegit.org/guides/cloning/#clone-options

> Unfortunately in OS X there is a problem where libgit2 is unable to look up GitHub certificates correctly. In order to bypass this problem, we’re going to passthrough the certificate check.